describe.go

```diff
@@ -43,6 +43,12 @@ func Describe(message *Message, w io.Writer) error {
 	// display the rest of all set fields
 	fields := message.GetFields()
 
+	// making filter map
+	filterMap := make(map[int]FilterFunc)
+	for _, filter := range filters {
```

Comment:
can we here do following:
* if no filters are passed - use DefaultFilters
* otherwise - use passed filters

If we do so, all users will get "safe by default" output. If they don't want to filter fields and want to get unfiltered data, then let's create for them no-op filter so they can:

```
Describe(message, os.Stdout, iso8583.DoNotFilterFields)
```

Reply:
Sounds good to me.